shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1146,7 +1245,7 @@ public void process(RequestProcessor cp) {
         public void writeFields(DataOutput out) throws IOException {
             out.writeLong(requestId);
             out.writeUTF(segment);
-            out.writeUTF(delegationToken == null ? "" : delegationToken);
+            out.writeUTF(delegationToken);
```

Comment:
Why this change? The Javadoc says this will throw an NPE if it's null.

Reply:
removed this unnecessary change.